api/views.py

```diff
@@ -80,3 +81,11 @@ class UserFeedbackViewSet(ModelViewSet):
 
     def perform_create(self, serializer):
         serializer.save(reported_by=self.request.user)
+
+
+class AssetStatusViewSet(ModelViewSet):
+    serializer_class = AssetStatusSerializer
+    queryset = AssetStatus.objects.all()
+    permission_classes = [IsAuthenticated]
+    authentication_classes = (FirebaseTokenAuthentication,)
```

Comment:
For consistency, let's have `authentication_classes = [FirebaseTokenAuthentication,] ` since we have already used a list to describe the `permission_classes`. Also, add a comma inside the permission_classes, so it looks like `permission_classes = [IsAuthenticated,]`